book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] To create a rental here we need `person` and `date` and the `book` itself should be sent as a parameter to create the new rental achieving the 'has-many' association
```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```